splice_machine/src/main/java/com/splicemachine/derby/stream/function/csv/QuoteTrackingTokenizer.java

```diff
@@ -310,8 +314,10 @@ private String getCurrentLine() {
             return currentRow.last();
         }
     }
+    boolean is0D0Amode = false;
 
```

Comment:
is0D0Amode is always false and never true, then why we need this flag?

Reply:
you're right, that's a leftover from a previous attempt i forgot to remove. thanks!